### ISSUE TITLE
Installation of Nunaliit

### ISSUE THREAD

**Issue by wanderer33third** [GCRC/nunaliit#436]
I am unable to finish installing Nunaliit (on Mac OSX El Captian).  I have downloaded the most recent tar.gz file from Maven, and have downloaded and installed all of the prerequisite programs (CouchDB, ImageMagick, etc.).  I have uncompressed the Nunaliit tar.gz (2.2.5).  I do not know how to proceed from this point, and the documentation on the wiki (under "Installing Nunaliit") is unclear to me.  I am a novice with command-line programs, and would appreciate any assistance in completing the installation.

**Comment by jpfiset** [comment]
Were you able to complete the steps mentioned here?

https://github.com/GCRC/nunaliit/wiki/Installation#installing-nunaliit

**Comment by wanderer33third** [comment]
I was able to complete everything except for the section 'Installing Nunaliit.'  I tried entering the commands in the text box (below) in my terminal window in the same way I installed some of the prerequisite programs, and it did not recognize the commands.  

cd /usr/local
tar -zxvf nunaliit2-couch-sdk-X.Y-nunaliit.tar.gz
chmod -R o+r,o+x nunaliit2-couch-sdk-X.Y-nunaliit
export PATH=$PATH:/usr/local/nunaliit2-couch-sdk-X.Y-nunaliit/bin

I also tried opening the nunaliit file within the bin folder of the uncompressed Nunaliit package that I downloaded, and it gave me an error (see below)

2016-01-26 11:26:15,620[ERROR]: Error: No command provided. Try 'help'.
java.lang.Exception: No command provided. Try 'help'.
	at ca.carleton.gcrc.couch.command.Main.execute(Main.java:119)
	at ca.carleton.gcrc.couch.command.Main.main(Main.java:57)
Error: No command provided. Try 'help'.
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.
Deleting expired sessions...none found.

[Process completed]
 
As I said, I am a novice using this type of program, so I may just be missing a step that those who are more experienced would have recognized in the instructions provided, but I am unable how to proceed.  Any assistance would be greatly appreciated

**Comment by pulsifer** [comment]
Hi,

I am not a developer, but have been using Nunaliit for some time, so may be able to help.

If I recall, the nunaliit command requires an argument.  for example:

/usr/local/nunaliit2-couch-sdk-X.Y-nunaliit/bin/nunaliit help

OR simply 

nunaliit help (assuming that you PATH statement is working)

As a start, you can try this and see if you are presented with more meaningful output.  I will look for follow up messages.

**Comment by wanderer33third** [comment]
Since many of the error messages I was receiving were some form of "file or directory not found", the comment 'assuming that your PATH statement is working' made me think there was something I was missing about using the command line.  

I ended up finding this website - http://coolestguidesontheplanet.com/add-shell-path-osx/ - which helped me figure out how to change my PATH (like I said, huge novice).

My problem was that I was not changing the 'usr/local' after cd to my actual PATH, and had not changed the file names (specifically the 'X.Y' placeholder for the file version) to the filename that I had downloaded.  Once I realized how these parts needed to be changed (through trial and error and how-to websites) I was able to finish the installation.  Or at least I have gotten to the point where I am able to move into the 'Using Nunaliit' section of the Atlas Builder Documentation where I can start creating a test atlas.  

Thank you very much for everyone's assistance
